recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-module-egl-headers.inc

```diff
@@ -17,8 +17,10 @@ do_configure() {
 
 do_install() {
     # Install header files
-    install -d ${D}/${includedir}/CL
-    install -m 644 ${S}/include/khronos/CL/*.h ${D}/${includedir}/CL/
+    if [ -d ${S}/include/khronos/CL ]; then
+        install -d ${D}/${includedir}/CL
```

Comment:
Seems you can move first "install" out of the "if". No?

Reply:
yes, but for clarity I would prefer not to see empty CL if it is not present
it may mislead

Reply:
I see

Reply:
any comments..?